heuristic/classes/Solution.py

```diff
@@ -165,3 +165,38 @@ def to_file(self, location: str):
                     print(f"V{idx_route},{leg},S{idx_stack},{stack}", file=file)
 
         file.close()
+
+    def to_graph(self):
+        problem = Problem()
+
+        import networkx as nx
+        import matplotlib.pyplot as plt
+        import numpy as np
+        from sklearn.decomposition import PCA
+        from sklearn import manifold
```

Comment:
These should go up (top of the file). Make sure to add networkx and sklearn to the Pipfile, too!